doc/api/repl.md

```diff
@@ -401,6 +401,17 @@ deprecated: REPLACEME
 An internal method used to parse and execute `REPLServer` keywords.
 Returns `true` if `keyword` is a valid keyword, otherwise `false`.
 
+### replServer.turnOffEditorMode()
+<!-- YAML
+added: v6.4.0
+deprecated: REPLACEME
+-->
+
+> Stability: 0 - Deprecated.
+
+An internal method used to turn off editor mode when entering
+code in the REPL.
+
```

Comment:
Actually I think we do not have to document this at all. This would create bigger confusion than providing benefit.

Reply:
Hmm - I would agree, but previous similar deprecations have been handled in this way. 

See:
* https://github.com/nodejs/node/issues/7619#issuecomment-236608809
* https://github.com/nodejs/node/blob/master/doc/api/deprecations.md#dep0074-replserverbufferedcommand.

Reply:
I think documenting this is the opposite of what we actually want to achieve. I do not really read in that comment that we necessarily have to document this either and I would rather only have the deprecation notice that this was never meant to be exposed and that it is now deprecated but no further documentation.

@jasnell what do you think about this?

Reply:
By the way - the deprecation docs to not cover this part at all. Therefore I think it is safe to remove this.

Reply:
@BridgeAR are you suggesting removing [this commit](https://github.com/nodejs/node/pull/15136/commits/7e9e7eab99494bbdd569d53b8dbaa919d0268a6c) entirely? Or just removing the wording from `repl.md`?

Reply:
Only the part from the `repl.md`.